pkg/compliance/report.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package compliance
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/compliance/event"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// Report contains the result of a compliance check
+type Report struct {
+	// Data contains arbitrary data linked to check evaluation
+	Data event.Data
+	// Passed defines whether check was successful or not
+	Passed bool
+}
+
+// BuildReportForUnstructured returns default Report for Kubernetes objects
```

Comment:
Nit: maybe split it to a separate file, it's somewhat specific to Kubernetes and might confuse someone just looking to understand the structure of the `Report` type.

Reply:
Move to `report_utils.go` though not great name either